### PR TITLE
fclones: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/fclones.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Busca ficheros duplicados en el directorio actual:
+
+`fclones group .`
+
+- Busca archivos duplicados en varios directorios y almacena los resultados en caché:
+
+`fclones group --cache {{ruta/a/directorio1 ruta/a/directorio2}}`
+
+- Busca archivos duplicados sólo en el directorio especificado, omitiendo los subdirectorios, y guardar los resultados en un archivo:
```
Comment (realFranco) [inline pages.es/common/fclones.md:14]:
```suggestion
- Busca archivos duplicados sólo en el directorio especificado, omitiendo los subdirectorios y guarda los resultados en un archivo:
```

**Review by realFranco — CHANGES_REQUESTED** [review]

File: pages.es/common/fclones.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`fclones group {{ruta/a/directorio}} --depth 1 > {{ruta/al/archivo.txt}}`
+
+- Mueve los archivos duplicados en un archivo TXT en un directorio diferente:
+
+`fclones move {{ruta/a/directorio_objetivo}} < {{ruta/al/archivo.txt}}`
+
+- Realiza un simulacro de enlaces blandos en un archivo TXT sin enlazarlo realmente:
+
+`fclones link --soft < {{ruta/al/archivo.txt}} --dry-run 2 > /dev/null`
```
Comment (tricantivu) [inline pages.es/common/fclones.md:24]:
```suggestion
`fclones link --soft < {{ruta/al/archivo.txt}} --dry-run 2> /dev/null`
```

I am not sure if the option `--dry-run` accepts an argument. If not, the redirection is erroneous.

The example is very similar to the [last one in this section of the project's README](https://github.com/pkolaczk/fclones/blob/main/fclones/README.md#removing-files)

**Comment by realFranco** [comment]
> Do you think we should write `archivo de texto` instead of `archivo TXT` @kant @realFranco?
> 
> I think the former might be slightly ambiguous.

Hello @tricantivu thank you for your contribs.

About your question, I will opt for `archivo de texto` instead of include `TXT` on the example description.

File: pages.es/common/fclones.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Busca archivos duplicados en varios directorios y almacena los resultados en caché:
+
+`fclones group --cache {{ruta/a/directorio1 ruta/a/directorio2}}`
+
+- Busca archivos duplicados sólo en el directorio especificado, omitiendo los subdirectorios, y guardar los resultados en un archivo:
+
+`fclones group {{ruta/a/directorio}} --depth 1 > {{ruta/al/archivo.txt}}`
+
+- Mueve los archivos duplicados en un archivo TXT en un directorio diferente:
```
Comment (tricantivu) [inline pages.es/common/fclones.md:18]:
```suggestion
- Mueve los archivos duplicados en un archivo de texto en un directorio diferente:
```

**Review by kant — COMMENTED** [review]
LGTM

**Review by realFranco — APPROVED** [review]

**Comment by realFranco** [comment]
LGTM

**Review by tricantivu — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**